src/pages/item/item-actions/itemActions.cy.tsx

```diff
@@ -5,7 +5,60 @@ const mockItemId = 201284;
 
 describe('<ItemActions>', () => {
 	it('mounts', () => {
-		cy.mountWithStore(<ItemActions itemId={mockItemId} />);
+		cy.mountWithStore(<ItemActions itemId={mockItemId} cardId={''} />);
```

Comment:
Tell me when the tests are ready. (You could "Re-request" then a review I think)

Reply:
pushed finished tests